### PR TITLE
Not rebooting if debug level > 0 [full ci]

### LINKED ISSUE #1999: VCH should not reboot endlessly on panic
Currently if the vic-init process (which is the default process on the VCH) panics, we reboot the VCH. This is not useful at all because we need to look at the logs for the panic call stack, etc.; these are obliterated on reboot.

We need to:
1. stop rebooting on panic
2. launch vic-admin as a separate (systemd) service so that it is always running on the VCH, even if vic-init crashes

### PR DESCRIPTION
**Pull request by hmahmood**
Fixes #1999

### CODE REVIEW

**Comment by sgairo** [comment]
LGTM

File: pkg/trace/operation_test.go [hunk 1]
```diff
@@ -103,8 +107,8 @@ func TestNestedLogging(t *testing.T) {
 
 	// We arrive at 2 because we have the err line (line 0), then the root
 	// (line 11) of where we created the ctx.
-	if !assert.True(t, lines >= levels+2) {
-		t.Logf("exepected %d and got %d", levels+2, lines)
+	if !assert.True(t, lines >= levels) {
```
Comment (anchal-agrawal) [inline pkg/trace/operation_test.go]: Can we use `assert.False` instead of `!assert.True`?

**Comment by anchal-agrawal** [comment]
LGTM

File: pkg/trace/operation_test.go [hunk 2]
```diff
@@ -103,8 +107,8 @@ func TestNestedLogging(t *testing.T) {
...
+		t.Logf("exepected %d and got %d", levels, lines)
```
Comment (anchal-agrawal) [inline pkg/trace/operation_test.go]: `expected`

File: cmd/vic-init/main_linux.go [hunk 3]
```diff
@@ -44,6 +47,17 @@ func main() {
 		reboot()
 	}()
 
+	src, err := extraconfig.GuestInfoSourceWithPrefix("init")
+	if err != nil {
+		log.Error(err)
```
Comment (vburenin) [inline cmd/vic-init/main_linux.go]: Is it an error that we don't care? Should not it be a warning?

Comment (vburenin) [inline cmd/vic-init/main_linux.go]: I see, it belongs to main, not to init. nvm.

**Comment by anchal-agrawal** [comment]
lgtm

File: pkg/trace/trace.go [hunk 4]
```diff
@@ -16,21 +16,13 @@ package trace
...
 
-var Logger = &log.Logger{
-	Out: os.Stderr,
-	// Apparently this forces terminal handling even when Out isn't a TTY (like
-	// on a VCH, where STDOUT/STDERR are currently redirected to a file).
-	Formatter: &log.TextFormatter{ForceColors: true},
-	Hooks:     make(log.LevelHooks),
-	Level:     log.InfoLevel,
-}
+var Logger = logrus.New()
```
Comment (caglar10ur) [inline pkg/trace/trace.go]: I think we need @fdawg4l eyes on this as he added this couple of days ago

Comment (fdawg4l) [inline pkg/trace/trace.go]: Yeah, this breaks our logging.  `logrus` does a syscall on the fd for STDOUT to check if it's a tty and if it is not, it blindly passes escape sequences like line-wrapping.    Yes, really.  If you want to verify, startup a VCH, and tail the port layer log and you'll see all of the `Operation` log messages will include their "\n"s and "\t"s when there is a stack trace.  Ergo the comment in the original version of this file.

Comment (fdawg4l) [inline pkg/trace/trace.go]: @hmahmood did the unit test pass on drone?  It would fail in very much the same way on the VCH.

Comment (fdawg4l) [inline pkg/trace/trace.go]: Also why do you need this?

**Comment by hmahmood** [comment]
♻️

**Pull request merged**